src/components/campaigns/ViewCampaignPage.tsx

```diff
@@ -49,7 +49,11 @@ export default function ViewCampaignPage({ slug }: Props) {
   const ogImageUrl = campaignListPictureUrl(campaign)
 
   return (
-    <Layout maxWidth={false} ogImage={ogImageUrl} metaDescription={campaign.title}>
+    <Layout
+      maxWidth={false}
+      ogImage={ogImageUrl}
+      title={campaign.title}
```

Comment:
I tried this before, but it has a side effect of the title showing over the campaign image too, so it looks like more changes are needed?

Reply:
@igoychev Thanks for mentioning! 

![image](https://user-images.githubusercontent.com/893608/183076790-a485cea5-8b23-4fe9-8b71-fef89b31e795.png)


Indeed it looks inadequate, will find a solution

Reply:
@igoychev 

I've added the option to pass only `metaTitle` and `title` separately:

```tsx
<Layout
  title="campaign.title"
  metaTitle="meta.title"
/>
```

![image](https://user-images.githubusercontent.com/893608/183078254-416c6153-6c7e-4afb-a07c-e9d83fd1d6a1.png)